.github/actions/integration-tests/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Integration tests'
+description: 'Runs integration tests'
+inputs:
+  upgrade-image-name:
+    description: 'Image that will api-gateway be upgraded to'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-go@v4
+      with:
+        go-version: "1.20"
+    - name: Add local.kyma.dev to /etc/hosts
+      shell: bash
+      run: |
+        sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
+    - name: Run integration tests
+      env:
+        KYMA_DOMAIN: "local.kyma.dev"
+      run: |
```

Comment:
In the `Makefile` we have target installing kyma CLI binary locally. We should consider reusing it (not necessarily now)